docs/docs-book/src/using-coffee.md

```diff
@@ -3,44 +3,111 @@
 Coffee is a plugin manager and development tool for core lightning nodes. It helps automate configuration and installation of plugins, and provides plugin templates for all the official languages.
 
 Coffee is a command line utility that provides access to a wide range of tasks.
+
 ## First Configuration
 
+> ✅ Implemented
+
+Coffe is able to manage the configuration of your core lightning and all the plugins connected with it, so
+in order to use coffe the user need to point coffee in the current home of core lightning.
+
+By default the core lightning home is stored in the `/home/<user>/.lightnig`, and you can do it with the
```

Comment:
```suggestion
By default the Core Lightning home is stored in the `/home/<USER>/.lightning`, and you can do it with the
```

Reply:
I do not like capitalize words like `USER`